.github/workflows/publish-canary.yml

```diff
@@ -9,8 +9,8 @@ on:
         description: 'Dist tag for the release. If you chose something different than "canary", make sure to delete it once it is not needed anymore.'
         type: string
         required: false
-        default: 'canary'
-
+        default: 'next'
+        
```

Comment:
[req] Just setting tag next is not enough for a next release. The version has to be 2.0.0. With the current setup 
```
cat <<EOT >> .changeset/canary-release-changeset.md
          ---
          '@sap-ai-sdk/core': patch
          ---
          Canary release
```
we'll end up publishing 1.0.0 as next which is misleading.

While we are in v2 development, we need to adjust is to use `major` instead of `patch`. Once the release is done it has to be reverted to use `patch`